packages/common-all/src/types/typesv2.ts

```diff
@@ -598,10 +598,25 @@ export enum DMessageSource {
   webClient = "webClient",
 }
 
+/**
+ * Basic mesages passed to the tree view
+ */
 export enum DMessageType {
+  // --- View Events
+  /**
+   * The view has just rendered
+   */
   INIT = "init",
-  ON_DID_CHANGE_ACTIVE_TEXT_EDITOR = "onDidChangeActiveTextEditor",
+  /**
+   * The view has listener for 
```

Comment:
nit: update comment to "The view is ready to accept messages from plugin dispatcher"? (If I remember correctly the purpose of this event)

Reply:
updated